### ISSUE TITLE
TokenMismatchException

### ISSUE THREAD

**Issue by fsuuaas** [harimayco/wmenu-builder#1]
VerifyCsrfToken.php (line 68) when create menu or add menu items in menu on laravel 5.5

**Comment by harimayco** [comment]
fixed on 1.0.1
please update your package
and then run
```php
php artisan vendor:publish --provider="Harimayco\Menu\MenuServiceProvider" --tag='view' --force
```
this command will override resources/views/vendor/harimayco-menu/menu.blade.php
Thanks

**Comment by fsuuaas** [comment]
menu.js:15 Uncaught TypeError: Cannot read property 'outerText' of undefined
    at HTMLLIElement.<anonymous> (menu.js:15)
    at Function.each (app.js:19)
    at yt.fn.init.each (app.js:19)
    at getmenus (menu.js:6)
    at HTMLAnchorElement.onclick (?menu=2:350)
